codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -411,6 +414,18 @@ private void applyLocalSettings(Http2Settings settings) throws Http2Exception {
             }
         }
 
+        /**
+         * Calculate the {@code maxStreams} paramter for the {@link Endpoint#maxStreams(int, int)} method based upon
+         * SETTINGS_MAX_CONCURRENT_STREAMS.
+         * @param maxConcurrentStreams SETTINGS_MAX_CONCURRENT_STREAMS
+         * @return the {@code maxStreams} paramter for the {@link Endpoint#maxStreams(int, int)} method.
+         */
+        @UnstableApi
+        protected int calculateMaxStreams(int maxConcurrentStreams) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this "protected" modifier. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2156)